src/components/Quiz/QuizRadioGroup.tsx

```diff
@@ -0,0 +1,164 @@
+          : iconBackgroundLight,
+      [showAnswer, isDarkMode]
+    )
+
+    // Render CustomRadio
+    return (
+      <chakra.label {...htmlProps} cursor="pointer" data-group w="100%">
+        <input {...getInputProps({})} hidden />
+        <Flex
+          {...getCheckboxProps()}
+          w="100%"
+          p={2}
+          alignItems="center"
+          bg={state.isChecked ? buttonBg : "quizButton"}
+          color={state.isChecked ? "white" : "text"}
+          borderRadius="base"
+          _hover={
+            showAnswer
+              ? {
+                  boxShadow: "none",
+                  cursor: "default",
+                }
+              : {
+                  outline: "1px solid var(--eth-colors-primary)",
+                  boxShadow: "primary",
+                }
+          }
+        >
+          <Circle
+            size={"25px"}
+            bg={circleBg}
+            _groupHover={
+              showAnswer
+                ? {
+                    bg: "white",
+                    cursor: "default",
+                  }
+                : {
+                    bg: isDarkMode ? "orange.700" : "blue.300",
+                  }
+            }
+            me={2}
+          >
+            <Text m="0" fontWeight={"700"} fontSize={"lg"} color={letterColor}>
+              {String.fromCharCode(97 + index).toUpperCase()}
+            </Text>
+          </Circle>
+          {label}
+        </Flex>
+      </chakra.label>
+    )
+  }
+
+  const handleChange = (value: string): void => {
+    handleSelection(value)
+  }
+
+  const { getRadioProps, getRootProps } = useRadioGroup({
+    onChange: handleChange,
+  })
+
+  // Memoized values
+  const explanation = useMemo<string>(() => {
+    if (!selectedAnswer) return ""
+    return answers.filter(({ id }) => id === selectedAnswer)[0].explanation
+  }, [selectedAnswer])
+  const letterColor = useMemo<string>(() => {
+    if (!showAnswer) return "white"
+    return correctAnswerId === selectedAnswer ? "green.500" : "red.500"
+  }, [questionData, selectedAnswer, showAnswer])
+
+  // Render QuizRadioGroup
+  return (
+    <Flex {...getRootProps()} direction="column" w="100%">
+      <Text fontWeight={"700"} fontSize={"2xl"} mb={6}>
+        {prompt}
+      </Text>
+      <Flex direction="column" gap={4}>
+        {answers.map(({ id, label }, index) => {
+          const display =
+            !showAnswer || id === selectedAnswer ? "inline-flex" : "none"
+          return (
+            <CustomRadio
+              key={id}
+              display={display}
+              index={index}
+              label={label}
+              {...getRadioProps({ value: id })}
+            />
+          )
+        })}
+      </Flex>
+      {showAnswer && (
+        <Box mt={5}>
+          <Text fontWeight="bold" mt={0} mb={2}>
+            Explanation
+          </Text>
+          <Text m={0}>{explanation}</Text>
+        </Box>
+      )}
+    </Flex>
+  )
+}
+
+export default QuizRadioGroup
```

Comment:
Awesome. Wondering now if we could cleanup and remove the `quizButton` variant from the `Button` theme.